notebooks/238-typo-detector/238-typo-detector.ipynb

```diff
@@ -0,0 +1,583 @@
+{
```

Comment:
<p>it is perfect, but you do not uses Infer Request directly in code below, so description is not complete or does not correspond of your inference way</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/908/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
Updated the description. Earlier I used Infer requests and when I changed it I forgot to update the description. Sorry about that. Please check now.